incubator/hnc/internal/reconcilers/test_helpers_test.go

```diff
@@ -195,6 +195,21 @@ func makeObjectWithAnnotation(ctx context.Context, kind string, nsName,
 	createdObjects = append(createdObjects, inst)
 }
 
+// updateObjectWithAnnotation gets an object given it's kind, nsName and name, adds the annotation
+// and updates this object
+func updateObjectWithAnnotation(ctx context.Context, kind string, nsName,
```

Comment:
Does it make more sense to have `makeObject` return an `*unstructured.Unstructured` and then you can just call `inst.SetAnnotations()` and `k8sClient.Update()` directly in the test function? I don't want to have _too_ many special-purpose helper functions.

Reply:
As discussed offline we decided to keep it as it is